### PR TITLE
fix docstring

### PR DESCRIPTION
**Pull request by st--**

### CODE REVIEW

**Review by theogf — APPROVED** [review]

File: src/latent_gp.jl [hunk 1]
```diff
@@ -2,9 +2,11 @@
     LatentGP(f<:GP, lik, Σy)
```
Comment (devmotion) [inline src/latent_gp.jl:2]: This should be

```suggestion
    LatentGP(f::AbstractGP, lik, Σy)
```

File: src/latent_gp.jl [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
 
  - `f` is a `AbstractGP`.
- - `lik` is the likelihood function which maps samples from `f` to the corresponding
- conditional likelihood distributions (i.e., `lik` must return a `Distribution` compatible with the observations).
- - `Σy` is the noise under which the latent GP is "observed"; this represents the jitter used to avoid numeric instability and should generally be small.
+ - `lik` is the likelihood function which maps samples from `f` to the
+   corresponding conditional likelihood distributions (i.e., `lik` must return a
+   `Distribution` compatible with the observations).
+ - `Σy` is the noise under which the latent GP is "observed"; this represents
+   the jitter used to avoid numeric instability and should generally be small.
```
Comment (devmotion) [inline src/latent_gp.jl:10]:
```suggestion
```

File: src/latent_gp.jl [hunk 3]
```diff
@@ -17,8 +19,9 @@ end
     LatentFiniteGP(fx<:FiniteGP, lik)
```
Comment (devmotion) [inline src/latent_gp.jl:19]:
```suggestion
    LatentFiniteGP(fx::FiniteGP, lik)
```

File: src/latent_gp.jl [hunk 4]
```diff
@@ -17,8 +19,9 @@ end
...
 
  - `fx` is a `FiniteGP`.
- - `lik` is the likelihood function which maps samples from `f` to the corresponding
- conditional likelihood distributions (i.e., `lik` must return a `Distribution` compatible with the observations).
+ - `lik` is the likelihood function which maps samples from `f` to the
+   corresponding conditional likelihood distributions (i.e., `lik` must return a
+   `Distribution` compatible with the observations).
```
Comment (devmotion) [inline src/latent_gp.jl:25]:
```suggestion
```

File: src/latent_gp.jl [hunk 5]
```diff
@@ -1,11 +1,12 @@
 """
-    LatentGP(f<:GP, lik, Σy)
+    LatentGP(f<:AbstractGP, lik, Σy)
```
Comment (devmotion) [inline src/latent_gp.jl:2]: This is still not correct, it seems?

**Pull request merged**